src/java/utils/JsonHelper.java

```diff
@@ -0,0 +1,17 @@
+package utils;
+
+import net.sf.json.JSONObject;
```

Comment:
This looks like a useful container. Have you let the rest of the team know they can use it?

Reply:
Yep, utils is a nice place for this kind of thing.  JsonHelper should be intuitive enough for developers.